Data/Graph/Inductive/PatriciaTree.hs

```diff
@@ -30,6 +30,7 @@ import Data.Graph.Inductive.Graph
 import           Control.Applicative (liftA2)
 import           Data.IntMap         (IntMap)
 import qualified Data.IntMap         as IM
+import qualified Data.IntMap.Strict  as IMS
```

Comment:
Yes. I think for some reason Cabal chooses a build plan for 7.4.2 that uses a version of containers older than 5.0. I think we can safely bump the containers lower bound to 5.0, which came out in 2012 and only demands a base version that came out in 2009.